src/logger.ts

```diff
@@ -102,7 +102,7 @@ export class Logger implements Disposable {
     }
 
     private log(level: vscode.LogLevel, scope: string, logToOutputChannel: boolean, args: any[]): void {
-        const msg = args.reduce((p, c, i) => {
+        const msg = args.reduce((p: string, c: any, i: number) => {
```

Comment:
type inference doesn't capture this already?

Reply:
It captured `c` and `i`, but not `p`, so `msg` ended up as `any`. I think the `+` with `any` confused the compiler.